lib/event_store_client/data_decryptor.rb

```diff
@@ -49,15 +49,8 @@ def deep_dup(hash)
     end
 
     def find_key(identifier)
-      key =
-        begin
-          key_repository.find(identifier).value!
-        rescue StandardError => e
-          config.error_handler&.call(e)
-          nil
-        end
-
-      key
+      key_repository.find(identifier).value!
+    rescue StandardError
```

Comment:
Where did the error handler go? We should not rescue an error and then do nothing.

Reply:
K, good. I wasn't sure about it.